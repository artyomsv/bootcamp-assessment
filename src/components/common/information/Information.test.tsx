import * as React from 'react';
import {createShallow} from '@material-ui/core/test-utils';
import Information from './Information.widget';

const shallow = createShallow();

describe('renders without crashing', () => {
  test('if renders', () => {
    const result = shallow(
      <Information
        label={'label'}
        value={'value'}
      />
    );

    expect(result).toMatchSnapshot();
  });
});
