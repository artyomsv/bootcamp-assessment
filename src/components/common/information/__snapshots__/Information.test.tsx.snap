// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing if renders 1`] = `
<Information
  classes={
    Object {
      "label": "Information-label-1",
      "row": "Information-row-3",
      "value": "Information-value-2",
    }
  }
  label="label"
  value="value"
/>
`;
